### ISSUE TITLE
What version of WCAG does ANDI comply to?

### ISSUE THREAD

**Issue by MFlintAHA** [SSAgov/ANDI#92]
I do not see in any of the documentation for ANDI that states what version of WCAG it references. I have found that 508 Standards incorporate by reference the WCAG 2.0 Level AA Success Criteria, so does that mean that ANDI also complies to WCAG 2.0 or is it referencing WCAG 2.1 Level AA success criteria?
With WCAG 3.0 being announced for 2023 are there plans to make ANDI reference to the latest rules?

**Comment by JohnCotterSSA** [comment]
Excellent questions!

ANDI is a manual test tool that should be used in combination with a manual test process. One example of a manual test process is the Trusted Tester method managed by DHS.  SSA has its own manual test process and has created ANDI to be the primary tool used in that process. Government agencies such as DHS and SSA are required to comply with Section 508 standards. Section 508 standards are currently based on WCAG 2.0 Level AA and A requirements. That will probably get updated as time goes on, and as long as SSA uses ANDI for 508 testing, SSA will want to keep ANDI up to date.

There is careful intention behind not directly referencing WCAG standards within ANDI. At the same time, many of ANDI's automated tests and manual assistance is based on WCAG 2.0 AA and A guidance. In some instances, ANDI also goes beyond  and throws some alerts and suggestions based on helpful WCAG 2.0 AAA guidelines, WCAG 2.1, and other best practices that are known to cause real issues to real people.

In general, ANDI's approach to accessibility testing is that the tester is the one to make the determination about what constitutes an accessibility issue. This allows ANDI users to use ANDI as an accessibility testing tool for their own organization's processes and business needs.
